extensions/amp-list/0.1/amp-list.js

```diff
@@ -366,6 +404,10 @@ export class AmpList extends AMP.BaseElement {
           promise = this.fetchList_();
         }
       } else if (typeof src === 'object') {
+        if (this.ssrTemplateHelper_.isEnabled()) {
+          user().error(TAG, '"[src]" may not be bound in SSR mode.');
```

Comment:
I think a `userAssert` is justifiable here to halt execution. We'd use `user().warn` if we wanted execution to continue.

Reply:
There are other equally erroneous conditions in which we currently allow execution to continue. I.e. the `'Unexpected "src" type` error. I'm not sure if it makes a difference?

Reply:
I think in that case there's nothing to render anyways.